src/libNode/FallbackBlockProcessing.cpp

```diff
@@ -292,8 +293,12 @@ bool Node::ProcessFallbackBlock(const bytes& message, unsigned int cur_offset,
         LOG_GENERAL(WARNING, "MoveUpdatesToDisk failed, what to do?");
         return;
       }
-      BlockStorage::GetBlockStorage().PutMetadata(MetaType::DSINCOMPLETED,
-                                                  {'0'});
+      if (!BlockStorage::GetBlockStorage().PutMetadata(MetaType::DSINCOMPLETED,
```

Comment:
How about move the error message into `PutMetadata` then we can handle the debug message in one place for easier maintenance. For caller here, it would be simpler to check the function result only.